system/keymaps/customcontroller.Harmony.xml

```diff
@@ -154,9 +154,6 @@
       <!-- right      	-->      <button id="104">StepForward</button>
       <!-- menu       	-->      <button id="106">OSD</button>
       <!-- Prev		-->      <button id="132">SmallStepBack</button>
-      <!-- Info  	-->      <button id="131">Info</button>
-      <!-- F7		-->      <button id="195">NextSubtitle</button>
```

Comment:
In global section the ids 195 and 194 are mapped differently. You can't remove them here bevause you'd loose the subtitle features that are mapped here

Reply:
But in global section the subtitle features are mapped to 174-175, which should be still available in the FullScreenVideo section.  I guess I don't understand how it works.

Reply:
I can't tell which buttons 194 and 195 are refering to. I just think that they were mapped for a reason in the first place.